src/App.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <link href="/dist/output.css" rel="stylesheet">
+  <ABVCalculator/>
```

Comment:
I would usually put a space after the Component name
Eg
`<ABVCalculator />`

Just makes it easier to read